pycbc/psd/analytical_space.py

```diff
@@ -184,32 +392,31 @@ def analytical_psd_lisa_tdi_2p0_XYZ(length, delta_f, low_freq_cutoff,
         The level of acceleration noise.
     oms_noise_level : float
         The level of OMS noise.
+    tdi : str
+        The version of TDI. Choose from "1.5" or "2.0".
 
     Returns
     -------
     fseries : FrequencySeries
-        The TDI-2.0 PSD (X,Y,Z channel) for LISA.
+        The TDI-1.5/2.0 PSD (X,Y,Z channel) for LISA.
     Notes
     -----
-        Please see Eq.(20) in <LISA-LCST-SGS-TN-001> for more details.
+        Please see Eq.(19-20) in <LISA-LCST-SGS-TN-001> for more details.
     """
-    len_arm = np.float64(len_arm)
-    acc_noise_level = np.float64(acc_noise_level)
-    oms_noise_level = np.float64(oms_noise_level)
     fr = np.linspace(low_freq_cutoff, (length-1)*2*delta_f, length)
-    s_acc_nu, s_oms_nu = lisa_psd_components(
-                          fr, acc_noise_level, oms_noise_level)
-    omega_len = omega_length(fr, len_arm)
-    psd = (64*(np.sin(omega_len))**2 * (np.sin(2*omega_len))**2 *
-           (s_oms_nu+s_acc_nu*(3+np.cos(2*omega_len))))
+    psd_components = np.array(lisa_psd_components(
+        fr, acc_noise_level, oms_noise_level))
```

Comment:
I have a minor question: what is the advantage of creating a numpy array and then unpacking them at, for example, line 361? My understanding is this can directly be transferred as a tuple, and then unpack like `s_acc_nu, s_oms_nu = psd_components`. (Or just set these two parameters as keyword arguments instead of `psd_components`)

Reply:
@HumphreyWang Thanks, that's just my bad habit, I have replaced all of them.